<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Park</class>
 <widget class="QMainWindow" name="Park">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Park</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>781</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>0</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>370</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>370</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>370</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>630</y>
      <width>781</width>
      <height>21</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>370</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_16">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>21</width>
      <height>281</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="midLineWidth">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="car">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nline">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mline">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="car_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>550</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>580</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>610</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>580</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_8">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>470</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_9">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>360</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_10">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>370</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_11">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>530</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_12">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>530</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_13">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>530</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_14">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>530</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="car_15">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>480</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>N</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Car No.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Money:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Money:</string>
    </property>
   </widget>
   <widget class="QLabel" name="tot_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="money_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="no_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>123456</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sec">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_min">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_hr">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_m233">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_722">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_q">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>72</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Queue:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="car_q">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>430</y>
      <width>81</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Empty:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_empty">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>72</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1188</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
